VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.html

```diff
@@ -68,6 +63,10 @@ <h3 class="govuk-heading-m">We're really sorry your hearing is delayed</h3>
           </p>
         </div>
 
+        <div *ngIf="hearing.isClosed()">
+          <h3 class="govuk-heading-m white">This hearing is finished. You may now sign out.</h3>
```

Comment:
The text to be 'Your hearing has finished.'

Reply:
fixed